lib/game.rb

```diff
@@ -124,8 +134,20 @@ def turn
           p  "Invalid coordinate, try again."     
           #display results of hits and misses every turn
         end
-        @computer_board.cells[coordinate.last].fire_upon
-        puts "Your shot on #{coordinate.first} was a #{shot_feedback(coordinate)}!"
+        #require 'pry'; binding.pry
+        #@computer_board.cells[coordinate].fired_upon? == false
+          @computer_board.cells[coordinate].fire_upon
+        # puts "Your shot on #{coordinate} was a #{shot_feedback(coordinate)}!"
+
+        if @computer_board.cells[coordinate].ship == nil
+          puts "Your shot on #{coordinate} was a miss!"
+        elsif @computer_board.cells[coordinate].ship == @com_submarine && @com_submarine.sunk? == true
```

Comment:
good work on these I realy like how you are covering all the bases with the conditionals